site/guides/porch-installation.md

```diff
@@ -69,7 +69,7 @@ To run a released version of Porch, download the release config bundle from
 Untar and applly the config bundle. This will install:
 
 * Porch server
-* [Config Sync](https://cloud.google.com/anthos-config-management/docs/config-sync-overview)
+* [Config Sync](https://kpt.dev/gitops/configsync)
```

Comment:
Shall this be "https://kpt.dev/gitops/configsync/" with the "/" at the tail end?  Otherwise it's broken link.